Polyhedron/demo/Polyhedron/MainWindow.cpp

```diff
@@ -860,6 +862,21 @@ void MainWindow::updateViewerBBox()
   qglviewer::Vec 
     vec_min(xmin, ymin, zmin),
     vec_max(xmax, ymax, zmax);
+  qglviewer::Vec offset(0,0,0);
+  for(int i=0; i<3; ++i)
+  {
+    if(std::log2(bbox.min(i)) > 13.0 )
+    {
+      offset[i] = -bbox.min(i);
+    }
+  }
+  if(offset != viewer->offset())
+  {
+    viewer->setOffset(offset);
+    recomputeItems();
```

Comment:
The strategy is strange. Instead of `std::log2(bbox.min(i))`, I would test something like the "distance" (l-infiinity distance) between the center of the bbox and the offset, something like:
```C++
max(abs(center.x() + offset.x()),
    max(abs(center.y() + offset.y()),
        abs(center.z() + oftset.z())))
```
and if that distance log2 is greater than 13 (half of the precision of `float`), then change the offset to the center of the bbox.

Using the `min` of the bbox as the offset is not the best choice, because it means that the sign-bit of the `float`s will never be used, and that wastes one bit of precision for no reason.